### PR TITLE
Mysql

### PR DESCRIPTION
**Pull request by jssuzanne**

### CODE REVIEW

**Comment by petrus-v** [comment]
Great ! testing anyblok against mysql ! looks like it'll require some extra effort to fix some tests

:clap: for that effort

File: anyblok/tests/test_column.py [hunk 1]
```diff
@@ -517,6 +517,8 @@ def test_datetime_by_query_filter(self):
         registry = self.init_registry(simple_column, ColumnType=DateTime)
         test = registry.Test.insert(col=now)
         Test = registry.Test
+        import ipdb
+        ipdb.set_trace()
```
Comment (petrus-v) [inline anyblok/tests/test_column.py]: I don't feel you wanted to include it in you commit ;)

Comment (petrus-v) [reply anyblok/tests/test_column.py]: ok forgot me I jump to a commit instead reading the whole diff

Comment (jssuzanne) [reply anyblok/tests/test_column.py]: it was a bad commit

**Pull request merged**